### DISCUSSION TITLE
How can we extract the individual random slopes from a MixedModel.jl fit?

### DISCUSSION THREAD

**Discussion by DarioSarra** [JuliaStats/MixedModels.jl#632] [Q&A] [answered]
This question is probably due to my lack of technical language understanding. 
I have a model m =  A ~ 1 + B + (1+B|S). I want the individual slopes of each S in respect of B. From the doc I read that `ranef` returns the conditional modes of the random effects given the observed data. An [online definition](https://stats.stackexchange.com/questions/214129/whats-the-interpretation-of-ranef-fixef-and-coef-in-mixed-effects-model-using) that I found explains that these are the differences between the (population-level) average predicted response for a given set of fixed-effect values (treatment) and the response predicted for particular individuals. 

So my understanding is that the coefficient of each S in respect of B can be calculated as the fixed effect of B .+ conditional modes. I implemented this in the following way:
`DataFrame(only(raneftables(m))).B .+ m.β[2]`

Is this conceptually and/or programmatically correct?

**Comment by palday** [comment, marked as answer]
Conceptually correct, definitely. I think the code is also correct, but I only looked at it very briefly.
